src/Databases/MySQL/DatabaseConnectionMySQL.h

```diff
@@ -4,17 +4,34 @@
 #if USE_MYSQL
 
 #include <mysqlxx/Pool.h>
+
+#include <Core/MultiEnum.h>
+#include <Common/ThreadPool.h>
 #include <Databases/DatabasesCommon.h>
-#include <memory>
 #include <Parsers/ASTCreateQuery.h>
-#include <Common/ThreadPool.h>
 
+#include <atomic>
+#include <condition_variable>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <unordered_set>
+#include <vector>
 
 namespace DB
 {
 
 class Context;
 
+enum class MySQLDataTypesSupport;
+
+std::map<String, NamesAndTypesList> fetchTablesColumnsList(
```

Comment:
Why do we declare it in here?

Reply:
Hmm... perhaps I should move it to `src/Core/MySQL/fetchTablesColumnsList.{h,cpp}`

Reply:
👍 Separate file is better.

Reply:
Done